#76 request

jascheng52 opened the pull request:
Added the option to lock the percent of the player and cpu

AlexanderHarrison commented:
Thanks for the PR! Looks great!

What's the reasoning for separating "CPU percent" and "CPU lock percent"? In my mind, they should be the same thing.

jascheng52 commented:
Sorry, but could specify which lines you are referring too? I am guessing that you meant the enum option i added in line 72 in lab.h. If so, I implemented it as a separate menu option that you can set on or off, and from my understanding to access the values(off or on in this case)you index the array based off the enum so it needs to be separated otherwise the offset would be off. I guess an alternative maybe would be a button toggle(like Y) instead that you click when hovering adjusting cpu percent to lock it and set a flag but I would have to look in on how to do that

AlexanderHarrison commented:
Oh my mistake, I read it wrong. I had thought the user had to manually set the locked percent separate from the other percent option.  

Looks good, thanks!
